qsignature/src/org/qcmg/sig/Options.java

```diff
@@ -93,6 +96,8 @@
 		.describedAs("hetLowerCutoff");
 		parser.accepts("noOfThreads", NO_OF_THREADS_OPTION_DESCRIPTION).withRequiredArg().ofType(Integer.class)
```

Comment:
This Long-form option name are not followed JP's docs rule. Here I think the option "--threads" is enough to indicate "number of threads".

Reply:
Out of scope for this PR